src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -382,6 +384,14 @@ public void registerCommitWebHook(@NonNull BitbucketWebHook hook) throws IOExcep
      * {@inheritDoc}
      */
     @Override
+    public void updateCommitWebHook(@NonNull BitbucketWebHook hook) throws IOException, InterruptedException {
+        putRequest(V2_API_BASE_URL + owner + "/" + repositoryName + "/hooks/" + Util.rawEncode(hook.getUuid()), JsonParser.toJson(hook));
```

Comment:
any encoding concerns on the repo name? (I have nightmares about encoding... so many scars)

Reply:
hrm no should be fine if BB allows it...